### PR TITLE
docs: re-word clarification about `setErrorHandler()`

### PR DESCRIPTION
**Pull request by codershiba**
#### Checklist
Follow-up PR to #5265 

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Reference/Errors.md [hunk 1]
```diff
@@ -169,11 +169,28 @@ Some things to consider in your custom error handler:
...
 [`setErrorHandler`](./Server.md#seterrorhandler),
-it's crucial to be aware of how errors 
+you should be aware of how errors 
 are propagated between custom and default error handlers.
 
+If a plugin's error handler re-throws
+an error, and the error is not an instance of 
+[Error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error)
+(as seen in the `/bad` route in the following example), it 
+would not propagate to the parent context error handler.
```
Comment (Fdawgs) [inline docs/Reference/Errors.md:181]:
```suggestion
(as seen in the `/bad` route in the following example), it 
will not propagate to the parent context error handler.
```

File: docs/Reference/Errors.md [hunk 2]
```diff
@@ -169,11 +169,28 @@ Some things to consider in your custom error handler:
...
+Instead, it will be caught by the default error handler.
+
+To ensure consistent error handling, 
+it is recommended to throw instances of Error.
+For instance, in the following example, replacing `throw 'foo'`
+with `throw new Error('foo')` in the
+`/bad` route would ensure that errors
```
Comment (Fdawgs) [inline docs/Reference/Errors.md:188]:
```suggestion
For instance, in the following example, replacing `throw 'foo'`
with `throw new Error('foo')` in the
`/bad` route ensures that errors
```

File: docs/Reference/Errors.md [hunk 3]
```diff
@@ -169,11 +169,28 @@ Some things to consider in your custom error handler:
...
+propagate through the custom error handling chain as intended.
+This practice helps to avoid potential
```
Comment (Fdawgs) [inline docs/Reference/Errors.md:190]:
```suggestion
This practice helps avoid potential
```

**Comment by codershiba** [comment]
Closed the PR by mistake and can't seem to re-open it ( the reopen button is greyed out )

**Pull request closed without merging**